YelpAPI.podspec

```diff
@@ -29,4 +29,5 @@ Pod::Spec.new do |s|
 
   s.public_header_files = 'Classes/**/*.h'
```

Comment:
Remove this; headers are public by default, but I'm not sure exactly what happens if you explicitly say they're public but then also explicitly say they're private (this conflicts with the `private_header_files` part)